optimism.tokenlist.json

```diff
@@ -847,6 +847,28 @@
       "symbol": "WETH",
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/logos/WETH.png"
+    },
+	{
+      "chainId": 42,
+      "address": "0x001dedC8c67EF6D1f4BB66FB28581d466650Cd76",
+      "name": "Dentacoin",
+      "symbol": "DCN",
+      "decimals": 0,
+      "logoURI": "https://ethereum-optimism.github.io/logos/logo.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0xc52D0B0627D80cf27DE27435362eF48f964d98e6"
```

Comment:
This should be the Optimism standard bridge on L1 kovan 0x22F24361D548e5FaAfb36d1437839f080363982B